WalletWasabi.Fluent/Views/AddWallet/TermsAndConditionsView.axaml

```diff
@@ -11,7 +11,7 @@
              x:DataType="vm:TermsAndConditionsViewModel"
              x:Class="WalletWasabi.Fluent.Views.AddWallet.TermsAndConditionsView">
   <c:ContentArea Title="{Binding Title}"
-                 Caption="To continue using Wasabi you must accept the terms and conditions"
+                 Caption="To continue using Wasabi please accept the terms and conditions"
```

Comment:
```suggestion
                 Caption="To continue using Wasabi, Please accept the terms and conditions."
```

Reply:
@yahiheb is actively working to remove the line ending `.` chars. I edited your suggestion to not add that.